app/views/layouts/_analytics.html.erb

```diff
@@ -1,70 +1,74 @@
 <% if Rails.application.config.analytics_tracking_id.present? %>
   <%= javascript_tag do %>
-   (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-                            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-   })(window,document,'script','https://www.google-analytics.com/analytics.js','ga')
+    var consentCookie = window.GOVUK.getConsentCookie();
 
-   ga('create', '<%= Rails.application.config.analytics_tracking_id %>', 'auto')
-   var dimensions = {}
+    if (consentCookie && consentCookie["usage"]) {
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+                              m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga')
 
-   $(document).ready(function() {
-     dimensions['dimension11'] = 'find-open-data'
-     ga('send', 'pageview', dimensions)
-     $('[data-ga-event]').on('click', sendAnalyticsEvent)
-   })
+      ga('create', '<%= Rails.application.config.analytics_tracking_id %>', 'auto')
```

Comment:
Looking at the output of the analytics in a browser extension, it looks like it's using a GA property id (tracking id) of `123456789`. This looks like a test value, just wanted to check that when it goes live it'll be set to something else?

Reply:
Yes, this is just a value I've set in the heroku app config as the code expect an environment variable to be set. When it's deployed, it will use the environment variable set in production.